notebooks/energy_volume_curve.ipynb

```diff
@@ -30,14 +30,28 @@
     "ExecuteTime": {
```

Comment:
<p>Great re-direct to related docs! But this is just pointing to the docs main page...the reader then needs to dig through and find the particular topic themselves? I think <a href="https://pyiron.readthedocs.io/en/latest/source/installation_remote.html" target="_blank">this</a> is a better page to point to</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Then I would just soften the language to something like “can be found i the pyiron docs, the homepage for which is here”</p>